src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlHelperPartialExtensions.cs

```diff
@@ -241,6 +241,94 @@ public static class HtmlHelperPartialExtensions
         /// <param name="partialViewName">
         /// The name of the partial view used to create the HTML markup. Must not be <c>null</c>.
```

Comment:
It's the same comment than in the Async version. I don't have a strong opinion about it, but I rather hear from more people before changing it.